widgets/contact/styles/default.less

```diff
@@ -84,6 +84,14 @@
 			}
 		}
 	}
+
+	.sow-form-field-description {
+		font-size: 0.9em;
+		margin-top: 0.2em;
+		margin-left: 0.2em;
+		color: #999;
```

Comment:
We should probably also have a field in the design section to change this default colour.